src/core/scientificNotation.ts

```diff
@@ -1,5 +1,5 @@
 function scientificNotationToDecimal(value: number | string): string {
-  if (!value) return "";
+  if (value == null) return "";
```

Comment:
Null is not type of value in StarkString internal value (_value) is always string